lib/gis/omp_threads.c

```diff
@@ -26,6 +31,10 @@ int G_set_omp_num_threads(struct Option *opt)
 
     int threads = atoi(opt->answer);
 #if defined(_OPENMP)
+    if (threads == 0) {
+        return omp_get_max_threads();
```

Comment:
This takes into account the OMP environmental variable?

Reply:
This does not attempt to change anything. omp_get_max_threads is called simply to return some number from the function. Theoretically it should match number of threads that will be used ([spec](https://www.openmp.org/spec-html/5.0/openmpsu112.html)).

Reply:
Yes, it takes into account the OMP_NUM_THREADS variable, if it is set. Generally `0` means – GRASS pushes all responsibility on to OpenMP implementation.